pages/docs/quickstart-forms.mdx

```diff
@@ -12,7 +12,7 @@ You can create your Profile by completing a Form on the BioDrop app.
 
 To do this, you need a GitHub account. If you do not have one yet, you can create one for free with an email address and password.
 
-1. Go to the [BioDrop homepage](https://biodrop.io/)
+1. Go to the <a href="https://biodrop.io/" target="_blank" rel="noopener">`BioDrop homepage`</a>
```

Comment:
Please use the `Link` component so the app so the code and visual display is kept consistent. Here is an example from another file in the docs https://github.com/EddieHubCommunity/BioDrop/blob/main/pages/docs/faqs.mdx#L14

Reply:
Yes, that a valid point.